### ISSUE TITLE
Not detecting comma and semicolon being mixed up

### ISSUE THREAD

**Issue by arioch** [rodjek/puppet-lint#13]
Lint does not detect the syntax error in this one:

```
package {
    'kvm':
        ensure => installed;
        name   => "${module::params::kvm}",
}
```
